stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -57,6 +58,10 @@ public void setPrettyPrinterInitializer(AsyncPrettyPrinterInitializer initialize
     mPrettyPrinterInitializer = Util.throwIfNull(initializer);
   }
 
+  public AsyncPrettyPrinterSchemaManager getPrettyPrinterSchemaManager() {
```

Comment:
Why not make this a singleton instance and just have `AsyncPrettyPrinterSchemaManager.getInstance()`